inc/class-sass.php

```diff
@@ -261,7 +261,11 @@ public function parseVariables( $scss ) {
 						$val = "{$fncall}({$fncall_params})";
 						break;
 					default:
-						$val = @( new \ScssPhp\ScssPhp\Compiler() )->compileValue( $item[2] ); // @codingStandardsIgnoreLine
+						try {
+							$val = @( new \ScssPhp\ScssPhp\Compiler() )->compileValue( $item[2] ); // @codingStandardsIgnoreLine
```

Comment:
This is the actual fix for broken integrations, definitely a refactor is needed here to avoid silencing errors with some nodes